models/articles.models.js

```diff
@@ -40,4 +40,37 @@ exports.fetchArticles = ()=>{
         return rows
     })
    
+}
+
+exports.fetchCommentsByArticleID = (article_id)=>{
+
+    const queryValues = []
+
+       
+     let sqlQuery = `SELECT  comments.comment_id, articles.article_id, comments.votes, comments.created_at, comments.author, comments.body
+    FROM comments
+    INNER JOIN articles ON comments.article_id = articles.article_id `
+    
+
+    if(article_id){
```

Comment:
don;t think this condition is necessary.  You are fetching by article ID and therefore should always have one.  So working with a complete SQL string should work fine.